apps/app/src/components/Volunteers/VolunteerCard.tsx

```diff
@@ -13,8 +13,17 @@ interface IVolunteerCardProps {
 }
 
 const VolunteerCard: React.FC<IVolunteerCardProps> = ({ post }) => {
+  let imageExtentions = ['.jpeg', '.jpg', '.png', '.gif', '.svg']
+  const validImage = (url: string) => {
+    let ok = false
+    imageExtentions.map((value) => {
```

Comment:
should be using .forEach instead of .map if you are not trying to transform the original array